src/components/VaultEditor/VaultEditor.tsx

```diff
@@ -0,0 +1,453 @@
+                                {(contracts.contracts || []).map((contract, index) =>
+                                    <ContractCovered
+                                        key={index}
+                                        contract={contract}
+                                        severitiesOptions={vaultDescription.severities.map(severity => ({
+                                            label: severity.name, value: severity.name
+                                        }))}
+                                        index={index}
+                                        contractsCount={(contracts.contracts || []).length}
+                                        onChange={onContractChange}
+                                        onRemove={removeContract}
+                                        addContract={addContract}
+                                    />)}
+                            </div>
+                        </div>
+                    </div>
+                </section>
+
+                <section className={classNames({ 'desktop-only': pageNumber !== 4 })}>
+                    <div className="vault-editor__section">
+                        <p className="vault-editor__section-title">
+                            5. {t("VaultEditor.pgp-key")}
+                        </p>
+                        <div className="vault-editor__section-content">
+                            <VaultProvider>
+                                <CommunicationChannel
+                                    removePgpKey={removePgpKey}
+                                    communicationChannel={vaultDescription?.["communication-channel"]}
+                                    addPgpKey={addPgpKey}
+                                    onChange={onChange}
+                                />
+                            </VaultProvider>
+                        </div>
+                    </div>
+
+                </section>
+
+                <div className="vault-editor__divider desktop-only"></div>
+
+                <section className={classNames({ 'desktop-only': pageNumber !== 5 })}>
+                    <div className="vault-editor__section">
+                        <p className="vault-editor__section-title">
+                            6. {t("VaultEditor.review-vault.title")}
+                        </p>
+                        <div className="vault-editor__section-content">
+                            <VaultReview vaultDescription={vaultDescription} />
+                        </div>
+                    </div>
+                </section>
+
+                <div className="vault-editor__button-container">
+                    {changed && ipfsHash && <button onClick={() => loadFromIpfs(ipfsHash)} className="fill">{t("VaultEditor.reset-button")}</button>}
+                    <button onClick={saveToIpfs} className="fill" disabled={!changed}>{t("VaultEditor.save-button")}</button>
+                </div>
+
+                {/* {
```

Comment:
Please remove comments

Reply:
i want to leave this code here for when we implement signing please